app/capabilities/capabilities.tsx

```diff
@@ -21,6 +22,8 @@ export class Capabilities {
     this.anonymous = window.buildbuddyConfig && window.buildbuddyConfig.anonymous_usage_enabled;
     this.enterprise = enterprise;
     this.paths = new Set(paths);
+    this.flameChart =
+      window.localStorage.getItem("buildbuddy://flags/flame_chart_enabled") === "true";
```

Comment:
I use this convention to organize stuff in localStorage because as the app grows and there gets to be a lot of stuff in localStorage, this hierarchical structure makes it easier to sort through the list in the Chrome console